kuma/wiki/models.py

```diff
@@ -811,6 +809,10 @@ def revert(self, revision, user, comment=None):
         revision.created = datetime.now()
         revision.creator = user
         revision.save()
+        # TODO: change to primary key check instead of object comparison
+        if revision.document.original == self:
+            revision.based_on = revision
+        revision.save()
```

Comment:
If the only thing changing between this save and the last one is the `based_on` FK, can we use `revision.save(update_fields=['based_on'])`?